sotodlib/observe/sim_sso.py

```diff
@@ -0,0 +1,271 @@
+# Copyright (c) 2018-2019 Simons Observatory.
+# Full license can be found in the top level "LICENSE" file.
+
+import os
+import numpy as np
+from toast.utils import Logger
+from toast.timing import function_timer, Timer
+from toast.op import Operator
+import ephem
+from toast.mpi import MPI
+import toast.qarray as qa
+import healpy as hp
+from scipy.constants import au as AU
+from scipy.interpolate import RectBivariateSpline
+import pickle
+
+def to_JD(t):
+    # Unix time stamp to Julian date
+    # (days since -4712-01-01 12:00:00 UTC)
+    return t / 86400.0 + 2440587.5
+
+
+def to_MJD(t):
+    # Convert Unix time stamp to modified Julian date
+    # (days since 1858-11-17 00:00:00 UTC)
+    return to_JD(t) - 2400000.5
+
+
+def to_DJD(t):
+    # Convert Unix time stamp to Dublin Julian date
+    # (days since 1899-12-31 12:00:00)
+    # This is the time format used by PyEphem
+    return to_JD(t) - 2415020
+
+
+def add_sim_sso_args(parser):
+    parser.add_argument(
+        "--simulate-sso",
+        required=False,
+        help="Comma-separated list of solar system objects (SSOs) to simulate",
+    )
+    parser.add_argument(
+        "--beam-file",
+        required=False,
+        help="Pickle file containing instrumental beam",
+    )
+    return
+
+
+def apply_sim_sso(args, comm, data, mc, totalname, verbose=True):
```

Comment:
It looks like this file is duplicate code that was split up into the other correct source files.  If so, can we delete it from this branch before merging?  Note that the `observe` directory does not even exist in the sotodlib package.

Reply:
Indeed! Thanks for catching this.